src/unstract/adapters/ocr/google_document_ai/src/static/json_schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "title": "Google Document AI OCR",
+  "type": "object",
+  "required": [
+    "adapter_name",
+    "url",
+    "credentials"
+  ],
+  "properties": {
+    "adapter_name": {
+      "type": "string",
+      "title": "OCR Adapter ID",
+      "default": "",
+      "description": "Provide a unique name for this adapter instance. Example: google-document-ai-1"
+    },
+    "url": {
+      "type": "string",
+      "title": "URL",
+      "default": "",
+      "format": "uri",
+      "description": "The URL of the Google Document AI endpoint for the processor Example: https://{endpoint}/v1/projects/{project}/locations/{location}/processors/{processor}"
+    },
+    "credentials": {
+      "type": "string",
+      "title": "Google Service Account",
+      "deafult": ""
+    }
```

Comment:
I think. It Will be better if we add asome description about what type of credential we expecting. Is it the json credentials of google account ?

Reply:
Done.